analyses/stackanalyses/controller.go

```diff
@@ -46,18 +48,17 @@ func StackAnalyses(requestParams driver.RequestType, jsonOut bool) bool {
 	mc.fileStats = mc.buildFileStats(requestParams.RawManifestFile)
 	postResponse := mc.postRequest(requestParams, mc.fileStats.DepsTreePath)
 	getResponse := mc.getRequest(requestParams, postResponse)
-	hasVul := summary.ProcessSummary(getResponse, jsonOut)
-	log.Debug().Msgf("Success StackAnalyses.")
-	return hasVul
-}
+	verboseEligible := getResponse.RegistrationStatus == RegisteredStatus
+	showVerboseMsg := verboseOut && !verboseEligible
 
-// GetManifestFilePath sets file path
-func (mc *Controller) GetManifestFilePath(input string) string {
-	path, err := filepath.Abs(input)
-	if err != nil {
-		log.Fatal().Msgf("Invalid Path of Manifest file. Only Absolute path is allowed.")
+	if verboseOut && verboseEligible {
+		hasVul = verbose.ProcessVerbose(getResponse, jsonOut)
+	} else {
+		hasVul = summary.ProcessSummary(getResponse, jsonOut, showVerboseMsg)
 	}
```

Comment:
It seems the behaviour fallbacks to summary Incase if user passes `--verbose`, but not eligible for verbose dump. Is this acceptable? I guess it is better to show the message to register with snyk and appropriate help command.

Reply:
@prashbnair , What is your opinion on this?

Reply:
This is what Parag wants, I guess we can ask them to register to see the complete information.

Reply:
Exactly